Gulpfile.js

```diff
@@ -0,0 +1,29 @@
+var gulp = require('gulp');
```

Comment:
**Code Quality: Nitpick**

I believe this file should be lowercase `gulpfile.js`.